compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -1563,25 +1563,27 @@ impl<'a, 'll, 'tcx> Builder<'a, 'll, 'tcx> {
         fn_abi: Option<&FnAbi<'tcx, Ty<'tcx>>>,
         llfn: &'ll Value,
     ) -> Option<llvm::OperandBundleDef<'ll>> {
-        let is_indirect_call = unsafe { llvm::LLVMIsAFunction(llfn).is_none() };
-        let kcfi_bundle = if is_indirect_call && self.tcx.sess.is_sanitizer_kcfi_enabled() {
-            if fn_attrs.is_some() && fn_attrs.unwrap().no_sanitize.contains(SanitizerSet::KCFI) {
-                return None;
-            }
+        let is_indirect_call = unsafe { llvm::LLVMRustIsNonGVFunctionPointerTy(llfn) };
+        let kcfi_bundle =
+            if is_indirect_call && fn_abi.is_some() && self.tcx.sess.is_sanitizer_kcfi_enabled() {
+                if fn_attrs.is_some() && fn_attrs.unwrap().no_sanitize.contains(SanitizerSet::KCFI)
```

Comment:
Nit: using `if let Some(fn_abi) = fn_abi` and `if let Some(fn_attrs) = fn_attrs` might be cleaner as you can avoid the later unwraps. Feel free to ignore.

Reply:
Changed. Thank you! :)